src/Microsoft.DotNet.Compiler.Common/BindingRedirectGenerator.cs

```diff
@@ -109,7 +109,11 @@ private static XElement GetOrAddElement(XContainer parent, XName elementName)
 
         private static AssemblyRedirect[] CollectRedirects(IEnumerable<LibraryExport> dependencies)
         {
-            var allRuntimeAssemblies = dependencies.SelectMany(d => d.RuntimeAssemblies).Select(GetAssemblyInfo).ToArray();
+            var allRuntimeAssemblies = dependencies
+                .SelectMany(d => d.RuntimeAssemblyGroups.GetGroup(string.Empty))
```

Comment:
I don't like passing string.empty around, it doesn't mean a lot. Maybe do optional parameter?